statement.go

```diff
@@ -134,8 +144,12 @@ func BuildStatement(input string) (Statement, error) {
 		matched := explainRe.FindStringSubmatch(input)
 		return &ExplainStatement{Explain: matched[1]}, nil
 	case showColumnsRe.MatchString(input):
-		matched := showColumnsRe.FindStringSubmatch(input)
-		return &ShowColumnsStatement{Table: matched[1]}, nil
+		matched := findStringSubmatchMap(showColumnsRe, input)
```

Comment:
Probably we can make this simpler?

1. Define regexp as `(?is)^(?:SHOW\s+COLUMNS\s+FROM)\s+(.+)$`.
2. Trim backtick (`) from matched string with [strings.Trim](https://golang.org/pkg/strings/#Trim).

Reply:
OK, is is reasonable because we don't care it is quoted identifier or not anymore.